source/cloud/cloud-administration/cloud-changelog.md

```diff
@@ -2,6 +2,35 @@
 
 This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/get-started/), an enterprise-grade SaaS offering hosted by Mattermost.
 
+## Release 2021-03-24
+
+### Improvements
+
+#### User Interface (UI)
+ - Added a UI field for restricted domains under **Site Configuration > Posts** in the System Console. Expanded the logic that determines whether a post has a preview or not.
+ - Added support for auto-RTL detection in browsers.
+ - Updated the font size for the **Add People** channel modal.
+ - Online status is now shown in the channel switcher.
```

Comment:
@amyblais May I have the feature PR driving this line too, please? Thanks!

Reply:
@cwarnermm https://github.com/mattermost/mattermost-webapp/pull/6609